storage/client.go

```diff
@@ -18,11 +18,23 @@ import (
 )
 
 const (
-	// DefaultBaseURL is the domain name used for storage requests when a
-	// default client is created.
+	// DefaultBaseURL is the domain name used for storage requests in the
+	// public cloud when a default client is created.
 	DefaultBaseURL = "core.windows.net"
 
-	// DefaultAPIVersion is the  Azure Storage API version string used when a
+	// ChinaBaseURL is the domain name used for storage requests in the
+	// China cloud when a client is created.
+	ChinaBaseURL = "core.chinacloudapi.cn"
```

Comment:
And then I realize that this is storage client -- maybe you don't want to add a new dependency on `go-autorest`.